docs/cookbook/payments/custom-invoice.rst

```diff
@@ -23,10 +23,34 @@ Before you start make sure that you have:
 #. `Sylius/InvoicingPlugin <https://github.com/Sylius/InvoicingPlugin/>`_ installed.
 #. `Wkhtmltopdf <https://wkhtmltopdf.org/>`_ package installed, because most of pdf generation is done with it.
 
+How to change the logo in the Invoice?
+----------------------------------------
+
+In order to change the logo on the invoice, set up the ``SYLIUS_INVOICING_LOGO_FILE`` environment variable.
+
+Example custom configuration:
+
+.. code-block:: text
+
+    SYLIUS_INVOICING_LOGO_FILE=/Users/sylius/logo_directory/your_custom_logo.png
```

Comment:
You should put a custom logo in the project directory and then use it with the proper path, so as to fix the problem from issue: https://github.com/Sylius/InvoicingPlugin/issues/254